care/facility/api/viewsets/patient.py

```diff
@@ -7,9 +7,11 @@
 from django_filters import rest_framework as filters
 from dry_rest_permissions.generics import DRYPermissionFiltersBase, DRYPermissions
 from rest_framework import serializers, viewsets
+from rest_framework.decorators import action
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework.decorators'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5346569)